tasks.py

```diff
@@ -0,0 +1,857 @@
+        run(
+            'test -r ~/.bash_profile && echo "eval \$($(brew --prefix)/bin/brew shellenv)" >>~/.bash_profile',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            'echo "eval $($(brew --prefix)/bin/brew shellenv)"',
+            hide=not verbose,
+            warn=True,
+        )
+        run(
+            "sudo apt-get install build-essential curl git", hide=not verbose, warn=True
+        )
+        if run("command -v brew", hide=not verbose, warn=True).ok:
+            run('echo "===>brew {}"'.format(MESSAGE_OK))
+        else:
+            run('echo "===>brew {}"'.format(MESSAGE_FAILED))
+
+    if PLATFORM == "Darwin":
+        if run("brew tap caskroom/cask", hide=not verbose, warm=True).ok:
+            run('echo "===>homebrew/cask is installed"')
+        else:
+            run('echo "===>homebrew/cask is not working."')
+
+    # creating a local setting file
+    if run('cd project/settings && touch local.py && echo "DEBUG=True" > local.py').ok:
+        run('echo "===>Successfully configured settings for local development"')
+    return None
+
+
+@task
+def install_dependencies(c, verbose=False):
+    """ install-dependencies invoke task. 
+
+    This func installs all the dependencies listed in the pipfile + djangosslserver.
+
+    Args:
+        c (obj): Context-aware API wrapper & state-passing object.
+        verbose (bool): states whether stdout should be printed.
+
+    Returns:
+        None.
+
+        However, this func echoes MESSAGE_FAILED, MESSSAGE_OK, or MESSAGE_ALREADY_INSTALLED
+        depending on the state of the installation process.
+
+    Note:
+        For debugging purposes, set verbose (bool) to True to print the stdout responses for the run process.
+
+    usage: 
+        invoke install-dependencies or invoke install-dependencies --verbose
+
+    """
+
+    run("echo '***INSTALLING ALL DEPENDENCIES***'")
+    packages = ("", "django-sslserver")
+    for package in packages:
+
+        if run("pipenv install {}".format(package), hide=not verbose, warn=True).ok:
+            run('echo "===>{} {}"'.format(package, MESSAGE_OK))
+        else:
+            run('echo "===>{} {}"'.format(package, MESSAGE_FAILED))
+    return None
+
+
+@task
+def pygraphviz(c, verbose=False):
+    """ pygraphviz invoke task.
+
+    This func installs graphyviz & pygraphviz.
+
+    Args:
+        c (obj): Context-aware API wrapper & state-passing object.
+        verbose (bool): states whether stdout should be printed.
+
+    Returns:
+        None.
+
+        However, this func echoes MESSAGE_FAILED, MESSSAGE_OK, or MESSAGE_ALREADY_INSTALLED
+        depending on the state of the installation process.
+
+    Note:
+        For debugging purposes, set verbose (bool) to True to print the stdout responses for the run process.
+
+    Usage: 
+        invoke pygraphviz or invoke pygraphviz --verbose
+
+    """
+
+    if PLATFORM == "Linux":
+        # Installing graphviz
+        """
```

Comment:
Go ahead and delete this stuff if it's not being used

Reply:
It's ok for this to be commented out for now but let's delete this if it's not being used